packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesUtils.java

```diff
@@ -171,7 +171,7 @@ public static String getRuntimeVersion(UpdatesConfiguration updatesConfiguration
     } else if (sdkVersion != null && sdkVersion.length() > 0) {
       return sdkVersion;
     } else {
-      throw new AssertionError("One of expo_runtime_version or expo_sdk_version must be defined in the Android app manifest");
+      return "1";
```

Comment:
comment added -- other places in this module assume this value will be nonnull, and for this edge case I think it's better to use a dummy value and show a clear warning/error in JS-land, rather than expect/handle a null value that really shouldn't ever be null.